python/lsst/meas/base/wrappers.py

```diff
@@ -116,9 +118,15 @@ def wrapAlgorithm(Base, AlgClass, factory, executionOrder, name=None, Control=No
         ConfigClass = wrapAlgorithmControl(Base.ConfigClass, Control, **kwds)
     def getExecutionOrder():
         return executionOrder
-    PluginClass = type(AlgClass.__name__ + Base.__name__, (Base,),
-                       dict(AlgClass=AlgClass, ConfigClass=ConfigClass, factory=staticmethod(factory),
-                            getExecutionOrder=staticmethod(getExecutionOrder)))
+    if TransformClass:
+        PluginClass = type(AlgClass.__name__ + Base.__name__, (Base,),
+                           dict(AlgClass=AlgClass, ConfigClass=ConfigClass, factory=staticmethod(factory),
+                                getExecutionOrder=staticmethod(getExecutionOrder),
+                                getTransformClass=staticmethod(lambda: TransformClass)))
+    else:
+        PluginClass = type(AlgClass.__name__ + Base.__name__, (Base,),
+                           dict(AlgClass=AlgClass, ConfigClass=ConfigClass, factory=staticmethod(factory),
+                                getExecutionOrder=staticmethod(getExecutionOrder)))
```

Comment:
Can't you avoid stuttering the common initialization code by building the dict beforehand and adding a key for getTransformClass only if necessary?